common/reviews/api/core.api.md

```diff
@@ -1393,8 +1431,7 @@ type ImageSpacingCalibratedEventDetail = {
     viewportId: string;
     renderingEngineId: string;
     imageId: string;
-    rowScale: number;

```

Comment:
Eventually row/column scale will be added back in as an affine transform, but doing it as scale tends to corrupt the annotation data in ways that cause problems.